packages/twenty-server/src/engine/core-modules/text-to-sql/text-to-sql.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+import { WorkspaceDataSourceModule } from 'src/engine/workspace-datasource/workspace-datasource.module';
+import { UserModule } from 'src/engine/core-modules/user/user.module';
+import { TextToSQLResolver } from 'src/engine/core-modules/text-to-sql/text-to-sql.resolver';
+import { TextToSQLService } from 'src/engine/core-modules/text-to-sql/text-to-sql.service';
+import { FeatureFlagEntity } from 'src/engine/core-modules/feature-flag/feature-flag.entity';
+@Module({
```

Comment:
One challenge would be to separate the "core ai engine" (which goes in engine- from "ask-ai" which seems to be more of a business module, as other apps will want to build on the same services. But as a first step it can be okay to keep it in one place if the distinction is to hard to make now